tests/sentry/issues/test_escalating.py

```diff
@@ -17,25 +17,21 @@
 from sentry.models import Group
 from sentry.models.group import GroupStatus
 from sentry.models.groupinbox import GroupInbox
-from sentry.testutils import TestCase
-from sentry.testutils.factories import Factories
+from sentry.testutils import SnubaTestCase, TestCase
```

Comment:
There are some changes in here inspired on #48228 and also to reduce conflict when either of the PRs merges.